pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -394,26 +428,49 @@ message Capability {
     repeated string drop_capabilities = 2;
 }
 
+// SecurityContext holds security configuration that will be applied to a container.
+message SecurityContext {
```

Comment:
Can we name this `ContainerSecurityContext`? I think the only reason it's not that way in the k8s API was that it was added before there was a `PodSecurityContext`